modules/node_modules/@frogpond/ccc-wpjson-feed/index.js

```diff
@@ -8,6 +8,13 @@ export async function fetchWpJson(url, query = {}) {
 }
 
 export function convertWpJsonItemToStory(item) {
+	let categories
+	if (item._embedded && item._embedded['wp:term']) {
+		categories = item._embedded['wp:term'].flatMap(cat =>
+			cat.filter(c => c.taxonomy === 'category').map(cat => cat.name),
```

Comment:
```suggestion
			cat.filter(c => c.taxonomy === 'category').map(c => c.name),
```

because I've already confused myself once about which variable the second `cat` was referring to.